pretty.js

```diff
@@ -51,9 +51,14 @@ function withSpaces (value, eol) {
   return lines.join(eol)
 }
 
-function filter (value, messageKey, eol) {
+function filter (value, messageKey, eol, isExcludeStandardKeys) {
```

Comment:
`isExcludeStandardKeys` would make more sense as `excludeStandardKeys`.

Reply:
rename to `excludeStandardKeys`